test/opt/workaround1209_test.cpp

```diff
@@ -0,0 +1,421 @@
+               OpCapability Shader
+          %1 = OpExtInstImport "GLSL.std.450"
+               OpMemoryModel Logical GLSL450
+               OpEntryPoint Vertex %main "main" %texcoord %gl_VertexIndex %_
+               OpSource GLSL 400
+               OpSourceExtension "GL_ARB_separate_shader_objects"
+               OpSourceExtension "GL_ARB_shading_language_420pack"
+               OpName %main "main"
+               OpName %texcoord "texcoord"
+               OpName %buf "buf"
+               OpMemberName %buf 0 "MVP"
+               OpMemberName %buf 1 "position"
+               OpMemberName %buf 2 "attr"
+               OpName %ubuf "ubuf"
+               OpName %gl_VertexIndex "gl_VertexIndex"
+               OpName %gl_PerVertex "gl_PerVertex"
+               OpMemberName %gl_PerVertex 0 "gl_Position"
+               OpName %_ ""
+               OpDecorate %texcoord Location 0
+               OpDecorate %_arr_v4float_uint_72 ArrayStride 16
+               OpDecorate %_arr_v4float_uint_72_0 ArrayStride 16
+               OpMemberDecorate %buf 0 ColMajor
+               OpMemberDecorate %buf 0 Offset 0
+               OpMemberDecorate %buf 0 MatrixStride 16
+               OpMemberDecorate %buf 1 Offset 64
+               OpMemberDecorate %buf 2 Offset 1216
+               OpDecorate %buf Block
+               OpDecorate %ubuf DescriptorSet 0
+               OpDecorate %ubuf Binding 0
+               OpDecorate %gl_VertexIndex BuiltIn VertexIndex
+               OpMemberDecorate %gl_PerVertex 0 BuiltIn Position
+               OpDecorate %gl_PerVertex Block
+       %void = OpTypeVoid
+         %12 = OpTypeFunction %void
+      %float = OpTypeFloat 32
+    %v4float = OpTypeVector %float 4
+%_ptr_Output_v4float = OpTypePointer Output %v4float
+   %texcoord = OpVariable %_ptr_Output_v4float Output
+%mat4v4float = OpTypeMatrix %v4float 4
+       %uint = OpTypeInt 32 0
+    %uint_72 = OpConstant %uint 72
+%_arr_v4float_uint_72 = OpTypeArray %v4float %uint_72
+%_arr_v4float_uint_72_0 = OpTypeArray %v4float %uint_72
+        %buf = OpTypeStruct %mat4v4float %_arr_v4float_uint_72 %_arr_v4float_uint_72_0
+%_ptr_Uniform_buf = OpTypePointer Uniform %buf
+       %ubuf = OpVariable %_ptr_Uniform_buf Uniform
+        %int = OpTypeInt 32 1
+      %int_2 = OpConstant %int 2
+%_ptr_Input_int = OpTypePointer Input %int
+%gl_VertexIndex = OpVariable %_ptr_Input_int Input
+%_ptr_Uniform_v4float = OpTypePointer Uniform %v4float
+%gl_PerVertex = OpTypeStruct %v4float
+%_ptr_Output_gl_PerVertex = OpTypePointer Output %gl_PerVertex
+          %_ = OpVariable %_ptr_Output_gl_PerVertex Output
+      %int_0 = OpConstant %int 0
+      %int_1 = OpConstant %int 1
+    %float_1 = OpConstant %float 1
+         %28 = OpConstantComposite %v4float %float_1 %float_1 %float_1 %float_1
+       %main = OpFunction %void None %12
+         %29 = OpLabel
+               OpBranch %30
+         %30 = OpLabel
+; CHECK: OpLoopMerge [[merge:%[a-zA-Z_\d]+]]
```

Comment:
FYI   you can use %\w+  as the pattern for a disassembled ID.   Effcee's RE2 pattern matcher is more expressive than FileCheck's